fixtures/showtec/performer-2500.json

```
@@ -0,0 +1,216 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Performer 2500",
+  "categories": ["Dimmer", "Color Changer"],
```

The colour temperature can be changed, but it's still white; I'd argue it's not really a colour changer. 
```suggestion
  "categories": ["Dimmer"],
```

---

Yes, I agree.